modules/faas/Makefile

```diff
@@ -22,11 +37,12 @@ faas/build/python/venv:
 faas/build/python/zip:
 	cd $(FAAS_BUILD_PYTHON_VENV) && zip $(FAAS_BUILD_DIST)/build.zip -rq *
 	cd func && zip $(FAAS_BUILD_DIST)/build.zip -r *py
+	cd $(FAAS_BUILD_DIST) && zip build.zip pyproject.toml
```

Comment:
Can this not go on the line above?

Reply:
don't think so - from what I know, `zip` places `pyproject.toml` in the archive tree relative to the current working directory. Since I want pyproject.toml to be at the root of the .zip archive, I need to cd into `dist/` to do so.